knowledge-base/animationcontainer-close-on-outside-click.md

```diff
@@ -50,7 +53,8 @@ To achieve the desired scenario:
 </TelerikAnimationContainer>
 
 @* suppress-error allows script tags in Razor files. Move this script to a separate file *@
-<script suppress-error="BL9992">//
+<script suppress-error="BL9992">
+    //
```

Comment:
We don't need these. I am using them locally to prevent VS for Mac breaking my code formatting.
```suggestion

```